nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/RunMongoAggregation.java

```diff
@@ -70,6 +70,10 @@
             .description("The result set of the aggregation will be sent to this relationship.")
             .name("results")
             .build();
+    static final Relationship REL_EMPTY = new Relationship.Builder()
```

Comment:
Should autoterminate by default because it's a new one on an existing processor, and we can't have people's flows suddenly breaking.

Reply:
Ok. Default autoterminate fixed.